core/src/translate/query.rs

```diff
@@ -181,34 +181,27 @@ fn translate_table_factor(sql_table_factor: &SqlTableFactor) -> Result<TableFact
         } => {
             let object_name = translate_object_name(name)?.to_uppercase();
             let alias = translate_table_alias(alias);
-            let alias_or_name = match &alias {
-                Some(alias) => alias.to_owned(),
-                None => TableAlias {
-                    name: object_name.to_owned(),
-                    columns: Vec::new(),
-                },
-            };
 
             match object_name.as_str() {
                 "SERIES" if args.is_some() => Ok(TableFactor::Series {
-                    alias: alias_or_name,
+                    alias: alias_or_name(alias, object_name),
                     size: translate_table_args(args)?,
```

Comment:
It seems that there is unreachable code inside the `translate_table_args` closure, because the value of `None` for args will never go into `translate_table_args`.